### ISSUE TITLE
Search Page not work - GLPI 9.4.4

### ISSUE THREAD

**Issue by gustavoandreotti** [glpi-project/glpi#6700]
**Describe the bug**

When you try to search something on "search_page" there is no result. 

**Page(s) URL**

/glpi/front/ticket.php

**To reproduce**

1. Go to 'Assistance' > 'Ticket'
2. Select to search all tickets
3. Click on 'Search'

**Expected behavior**

Show all tickets.

**Logs**

No error on logs.

 <img src="https://i.imgur.com/2MYTLAZ.png" alt="Print">

**Your GLPI setup (you can find it in Setup > General menu, System tab)**

```

General setup
--

General setup
--
Log Level | 1- Critical (login error only)2- Severe (not used)3- Important (successful logins)4- Notices (add, delete, tracking)5- Complete (all)5- Complete (all) | Maximal number of automatic actions (run by CLI) | 1010
Logs in files (SQL, email, automatic action...) | NoYesYes | SQL replica | NoYesNo
Password security policy
Password security policy validation | NoYesNo | Password minimum length | 88
Password need digit | NoYesYes | Password need lowercase character | NoYesYes
Password need uppercase character | NoYesYes | Password need symbol | NoYesYes
Maintenance mode
Maintenance mode | NoYesNo | Maintenance text |  
Proxy configuration for upgrade check
Server |   | Port |  
Login |   | Password | Clear
 


```
GLPI Version 9.4.4

**Comment by cedric-anne** [comment]
Hi,

It looks like page rendering has been interrupted. If it is not a PHP Fatal Error, you should have something into GLPI logs. If it is a PHP Fatal Error, you should have something into your system logs.

Without logs, is is nearly impossible to find what your problem is.

Regards

**Comment by cedric-anne** [comment]
Also, can you try with [nightly build](https://forge.glpi-project.org/tarballs/) to see if problem persist ?

**Comment by gustavoandreotti** [comment]
> Hi,
> 
> It looks like page rendering has been interrupted. If it is not a PHP Fatal Error, you should have something into GLPI logs. If it is a PHP Fatal Error, you should have something into your system logs.
> 
> Without logs, is is nearly impossible to find what your problem is.
> 
> Regards

Hello Cedric-anne, 

First of all, thank you so much for your reply. Unfortunelly there is no error logs on GLPI logs and PHP Apache too. 
I have changed the php.ini memory_limit = 64M to 128MB and the problem was solved, but when I try to check more than 18.000 tickets,  the same error ocorrurs.

Best Regards,

Gustavo

**Comment by cedric-anne** [comment]
Hi,

Currently, GLPI does not catch "memory_limit" errors, so they do not appear in GLPI logs (this may be done on 9.5 by #6726 ). The system configuration that have to be set to log this kind of errors into system logs depends on your system, so we cannot help for this.

If changing the memory limit fix this behaviour on a smaller subset of your ticket list, then your problem comes from your system configuration. You can either change the memory limit, either try to display less result on each page.

Regards
